.github/dependabot.yml

```diff
@@ -7,3 +8,13 @@ updates:
       # Check for updates to GitHub Actions every week on Saturday
       interval: "weekly"
       day: "saturday"
+  - package-ecosystem: "gomod"
+    directories:
+      - "src/cloud-api-adaptor"
+      - "src/cloud-api-adaptor/podvm"
+      - "src/cloud-providers"
+      - "src/csi-wrapper"
+      - "src/peerpod-ctrl"
+      - "src/webhook"
+    schedule:
+      interval: "daily"
```

Comment:
Do you think `daily` interval is really needed? How many people follow the main that closely? I think `monthly` would do, eventually we might force-update as part of release just to ensure we are releasing the latest bits, but testing daily seems like unnecessary overhead to me.

Reply:
> How many people follow the main that closely?

I don't understand what you mean here - we build the CI based on `main` every day, so all developers should be developing and testing against `main`. I'm not sure I understand the downsides of doing this more regularly unless some of our dependencies and getting updates super-frequently and understanding the reason for this would then be important to understand if we want to pick these up.

Reply:
I completely disagree that we should only be doing this just before release as that introduces all the risk into the time that we can least afford to cope with risk and delays. I'm happy to update to weekly as a compromise, but I'm strong against being less frequent than that cadence, so would need community consensus to overrule this.

Reply:
I agree that pre-release is the crucial time, which is why I suggested the (bi-)weekly. But daily seems a bit too much to me (but not a strong opinion so depends on others).